issue_template.md

```diff
@@ -0,0 +1,14 @@
+### Environment
+- Server - VSTS or TFS (provide version): 
+
+- Agent - Hosted or Private: 
+    - Agent queue name (if using Hosted agent): 
+    - Agent version and OS (if using Private agent): 
+
+### Issue Description
+
+
+### Debug logs
+> Run a build with variable `system.debug` set to `true` and attach the zip to the issue. If your logs have sensitive information, please sanitize them and paste relevant portion of the logs below.
```

Comment:
Minor suggestions:
- "attach the zip **of the logs** to **this** issue."
- "paste relevant **portions** of the logs"

Also, should we recommend **attaching** "relevant portions" instead of pasting them, which could be a lot of text?

Reply:
We want them to paste the error log here for SEO